app/src/main/java/sdp/moneyrun/Game.java

```diff
@@ -70,14 +67,28 @@ public Game(String gameId,
         this.isVisible = isVisible;
     }
 
-    public String getGameId(){
+
+    public static void endGame(List<Coin> collectedCoins, int playerId, Activity currentActivity) {
+        Intent endGameIntent = new Intent(currentActivity, EndGameActivity.class);
+        ArrayList<Integer> collectedCoinsValues = new ArrayList<>();
+        for (int i = 0; i < collectedCoins.size(); ++i) {
+            collectedCoinsValues.add(collectedCoins.get(i).getValue());
+        }
+        endGameIntent.putExtra("collectedCoins", collectedCoinsValues);
```

Comment:
Why not summing those values directly here ? I mean I don't see the advantage for the player to get all the separates values

Reply:
I did not code this function, I just moved it from one file to another.

Reply:
Ok then